Jobsway2goMvc/Controllers/EventsController.cs

```diff
@@ -113,6 +119,12 @@ public async Task<IActionResult> Create(Event @event, List<string> guestsIds)
                 _context.Notifications.Add(notification);
                 await _notificationHub.SendNotificationToClient(notification.Message, notification.UserName);
             }
+
+            var userid = _userManager.GetUserId(HttpContext.User);
+            ApplicationUser creator = _userManager.FindByIdAsync(userid).Result;
+            string uniqueFileName = UploadedFile(@event);
```

Comment:
`var userid` and `ApplicationUser creator` variables are redundant and cause errors. Remove them.